samples/.gitignore

```diff
@@ -0,0 +1 @@
+.logs/
```

Comment:
Adding this for sanity of everyone when testing the logging functionality. You **don't** want to check in the files elasticsearch creates 👍